### PR TITLE
Update current v2 setup

### PR DESCRIPTION
**Pull request by alexjoverm**
**Main features** of this whole update:
- Package is renamed to `@storyblok/vue` to scope the package under the storyblok org.
- Uses **Github Actions** for automatic building, testing and publishing
- **Docs** updated on the line that @christianzoppi worked on
- **Optimized build** with UMD and ES formats, supporting modern bundlers as well as plain load from a CDN
- Uses **npm workspaces** allowing to keep separated the lib and a playground. You can use `npm run dev` on the root to edit the library, having instant reload on the playground
- **Git hooks** in combination with Prettier, ESLint and Conventional Commit to make sure all code that goes to Github is consistently formatted and linted, and has valid commit messages.
- **Tests** are in place, both **unit** and **integration/e2e** using Jest and Cypress
- **Very lean** and **zero-config consistent structure** using to Vite. Meaning cleaner project, easier maintenance and faster development.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,20 +1,49 @@
...
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/storyblok/storyblok-vue"
-  },
-  "keywords": [
-    "vue",
-    "storyblok"
+  "name": "@storyblok/vue-ws",
+  "private": true,
+  "version": "0.0.1",
```
Comment (samuells) [inline package.json:4]: is this correct @alexjoverm ?

**Review by samuells — APPROVED** [review]
Please check the 2 questions I wrote. Other than that it looks 👍 Thx.

File: package.json (hunk 1 above)
Comment (alexjoverm) [reply package.json:4]: Not really relevant

File: README.md [hunk 2]
```diff
@@ -1,37 +1,105 @@
...
-https://jsfiddle.net/delooks/6tnttqvy/
+Where `blok` is the actual blok data coming from [Storblok's Content Delivery API](https://www.storyblok.com/docs/api/content-delivery?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-vue).
+
+Check out the [playground](/../../tree/master/playground) for a full example.
+
+### Compatibility
+
+This plugin is for Vue 2. Thus, it supports the [same browsers as Vue 2](https://vuejs.org/v2/guide/installation.html#Compatibility-Note).
+
+## 🔗 Related Links
```
[the remark this answers is not in the document]
Comment (alexjoverm) [reply README.md:85]: Nice idea

**Pull request merged**